core/embed/rust/src/ui/model_tt/component/page.rs

```diff
@@ -179,18 +245,60 @@ impl Component for ScrollBar {
             self.area.center().x,
             self.area.center().y - (count / 2) * interval,
         );
-        for i in 0..self.page_count {
-            display::rounded_rect(
-                Rect::from_center_and_size(dot, Self::DOT_SIZE),
-                if i == self.active_page {
-                    theme::FG
-                } else {
-                    theme::GREY_LIGHT
-                },
+        if self.has_previous_page() {
+            display::icon(
+                dot - Offset::new(0, Self::ARROW_SPACE),
```

Comment:
Maybe we should add `Offset::x()` and `Offset::y()` constructors for these cases?

Reply:
Definitely! Fixed in e0abc6c.